core/src/test/java/org/apache/iceberg/TableTestBase.java

```diff
@@ -162,6 +165,19 @@ public class TableTestBase {
 
   static final FileIO FILE_IO = new TestTables.LocalFileIO();
 
+  static final Map<String, String> CODEC_METADATA_MAPPING =
+      ImmutableMap.<String, String>builder()
+          .put("uncompressed", "null")
+          .put("zstd", "zstandard")
+          .put("gzip", "deflate")
+          .build();
+
+  static final String AVRO_CODEC_KEY = "avro.codec";
+
+  static final long SNAPSHOT_ID = 987134631982734L;
+
+  private static final long SEQUENCE_NUMBER = 34L;
```

Comment:
I would be fine with simply using the long value instead of defining this constant, as it is only used once.

Reply:
agreed, there doesn't seem too much value in having this in a constant field

Reply:
Removed.